runtime/solo/src/lib.rs

```diff
@@ -680,18 +675,17 @@ construct_runtime!(
 		CouncilMembership: pallet_membership::<Instance2> = 10,
 		Treasury: pallet_treasury = 11,
 		Democracy: pallet_democracy = 12,
-		Sudo: pallet_sudo = 13,
-		Scheduler: pallet_scheduler = 14,
-		Identity: pallet_identity = 15,
-		Proxy: pallet_proxy = 16,
-		Multisig: pallet_multisig = 17,
-		Utility: pallet_utility = 18,
-		Preimage: pallet_preimage = 19,
-		AwesomeAvatars: pallet_ajuna_awesome_avatars = 22,
-		Randomness: pallet_insecure_randomness_collective_flip = 23,
-		Nft: pallet_nfts = 24,
-		NftTransfer: pallet_ajuna_nft_transfer = 25,
-		NftStaking: pallet_ajuna_nft_staking = 26,
+		Scheduler: pallet_scheduler = 13,
+		Identity: pallet_identity = 14,
+		Proxy: pallet_proxy = 15,
+		Multisig: pallet_multisig = 16,
+		Utility: pallet_utility = 17,
+		Preimage: pallet_preimage = 18,
+		AwesomeAvatars: pallet_ajuna_awesome_avatars = 19,
+		Randomness: pallet_insecure_randomness_collective_flip = 20,
+		Nft: pallet_nfts = 21,
+		NftTransfer: pallet_ajuna_nft_transfer = 22,
+		NftStaking: pallet_ajuna_nft_staking = 23,
```

Comment:
I would recommend not changing the pallet-indices ever, as this is breaking change. However, I see that you have properly organized the pallets according to their category in the other runtimes. So I have two questions:

- Do you only use the solo node for dev purposes, or is the running solo network still a public test network?
- Why do you not apply the same pallet index scheme as in the bajun/ajuna runtimes?

Reply:
Heya! To answer your questions:

* Yeah, the solo node is only for dev purposes so its runtime is not as refined as the bajun/ajuna ones
* We don't mainly because the solo runtime being a test runtime has pallets added/removed from it more frequently and in general because being a test only node it's just less polished overall.

Reply:
Ok, I see this makes sense, however even in dev-nodes we have aligned the runtimes with the productions ones just to have as little discrepancy as possible to prevent unforeseen things also in rococo. Regardless, nothing to be solved here. Thanks for the answers!